test/views/govuk_wrapperSpec.scala

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Please rename file to match naming conventions of other test specs in the repo, i.e. `GovUkWrapperSpec`